UPGRADING.md

```diff
@@ -1,6 +1,10 @@
 Upgrading Slack-Ruby-Bot-Server
 ===============================
 
+### Upgrading to >= 0.8.3
+
+To avoid name collisions, `Team` has been renamed `SlackRubyBot::Team`, if you modify the class `Team` add `Team = SlackRubyBot::Team` before the code. 
```

Comment:
This should say `SlackRubyBotServer` not `SlackRubyBot`, right?

Reply:
ya, my bad. I'll get that changed.

Sure, I can test out MongoDB, but it might take a while though.